release notes/v0.4.0.md

```diff
@@ -0,0 +1,109 @@
+
+- Added `locator.isHidden` which will return `true` or `false` if an uniquely identifiable element is hidden ([#377](https://github.com/grafana/xk6-browser/pull/377))
+
+- Added `locator.fill` will fill an input with the supplied text ([#380](https://github.com/grafana/xk6-browser/pull/380))
+
+- Added `locator.focus` will focus on an uniquely identifiable element ([#388](https://github.com/grafana/xk6-browser/pull/388))
+
+- Added `locator.getAttribute` will return the named attribute of a selected locator ([#390](https://github.com/grafana/xk6-browser/pull/390))
+
+- Added `locator.innerHTML` which will return the html of the given selector ([#392](https://github.com/grafana/xk6-browser/pull/392))
+
+- Added `locator.innerText` will return the `innerText` from the locator ([#393](https://github.com/grafana/xk6-browser/pull/393))
+
+- Added `locator.textContent` will return the text from the locator ([#394](https://github.com/grafana/xk6-browser/pull/394))
+
+- Added `locator.inputValue` will return the text inputted in an input field ([#395](https://github.com/grafana/xk6-browser/pull/395))
+
+- Added `locator.selectOption` will select the given option in a select element ([](https://github.com/grafana/xk6-browser/pull/397))
+
+- Added `locator.press` which will "push" the specified keyboard key ([#398](https://github.com/grafana/xk6-browser/pull/398))
+
+- Added `locator.type` which will "type" the text into the input element ([#399](https://github.com/grafana/xk6-browser/pull/399))
+
+- Added `locator.hover` which will bring into view the selected element ([#400](https://github.com/grafana/xk6-browser/pull/400))
+
+- Added `locator.tap` which will bring the into view the selected element and tap on it ([#401](https://github.com/grafana/xk6-browser/pull/401))
+
+- Added `locator.dispatchEvent` which will dispatch the specified event to the selected element ([#405](https://github.com/grafana/xk6-browser/pull/405))
+
+- Added `locator.waitFor` which will wait for the element selected to meet the specified state ([#406](https://github.com/grafana/xk6-browser/pull/406))
+
+
+## Bugs fixed
+
+- Fixed zombie browser processes. Before this fix, browser processes remained active even after the test or k6 test runs. This could sometimes saturate the machine and therefore could affect the tests ([#306](https://github.com/grafana/xk6-browser/pull/306))
+
+- Fixed cleanup of the browser's temporary directory, so that it's properly removed when `Browser.close()` is called, or when the iteration or test run completes. ([#323](https://github.com/grafana/xk6-browser/pull/323), [#409](https://github.com/grafana/xk6-browser/pull/409))
+
+  This will help ensure that any private data that the browser caches to the temporary directory is deleted locally as well as when this is run in the cloud.
```

Comment:
Let's not mention the Cloud yet, at least not in this section. So maybe:
```suggestion
  This will help ensure that any private data that the browser caches in the temporary directory is deleted.
```